alarm/nialarm.yml

```diff
@@ -0,0 +1,686 @@
+        example: Low disk space on CRIO1
+      active:
+        type: boolean
+        example: true
+      clear:
+        type: boolean
+        example: false
+      acknowledged:
+        type: boolean
+        example: false
+      acknowledgedAtMin:
+        type: string
+        description: Acknowledged at minimum date query. The service will return instances which were acknowledged between *acknowledgedAtMin* and *acknowledgedAtMax*, inclusive.
+        example: '2019-01-14T23:15:53.7270539Z'
+      acknowledgedAtMax:
+        type: string
+        description: Acknowledged at maximum date query. The service will return instances which were acknowledged between *acknowledgedAtMin* and *acknowledgedAtMax*, inclusive.
+        example: '2019-01-15T23:15:53.7270539Z'
+      occurredAtMin:
+        type: string
+        description: Occurred at minimum date query. The service will return instances which occurred between *occurredAtMin* and *occurredAtMax*, inclusive.
+        example: '2019-01-14T23:15:53.7270539Z'
+      occurredAtMax:
+        type: string
+        description: Occurred at maximum date query. The service will return instances which occurred between *occurredAtMin* and *occurredAtMax*, inclusive.
+        example: '2019-01-15T23:15:53.7270539Z'
+      currentSeverityLevelMin:
+        type: integer
+        description: Current severity minimum query. The service will return instances whose current severity is between *currentSeverityLevelMin* and *currentSeverityLevelMax*, inclusive.
+        example: 1
+      currentSeverityLevelMax:
+        type: integer
+        description: Current severity maximum query. The service will return instances whose current severity is between *currentSeverityLevelMin* and *currentSeverityLevelMax*, inclusive.
+        example: 4
+      highestSeverityLevelMin:
+        type: integer
+        description: Highest severity minimum query. The service will return instances whose highest severity is between *highestSeverityLevelMin* and *highestSeverityLevelMax*, inclusive.
+        example: 1
+      highestSeverityLevelmax:
+        type: integer
+        description: Highest severity maximum query. The service will return instances whose highest severity is between *highestSeverityLevelMin* and *highestSeverityLevelMax*, inclusive.
+        example: 4
+      createdBy:
+        type: string
+        description: Created by query. The service will return instances whose *createdBy* fields are equal to the query, taking case into account.
+        example: TagRuleEngine
+      channel:
+        type: string
+        description: Channel query. The service will return instances whose *channel* fields are equal to the query, taking case into account. Supports wildcard match checking uses glob-style wildcards.
+        example: '*.System.Health.DiskSpaceUsePercentage'
+      properties:
+        type: object
```

Comment:
Should this have an additionalProperties attribute?

Reply:
Added missing additionalProperties attributes